### DISCUSSION TITLE
Auto generation of tokens for authentication

### DISCUSSION THREAD

**Discussion by mishrasur7** [Huachao/vscode-restclient#1199] [Q&A]
Does Rest Client support auto generation of tokens for making requests that require authorization?

**Comment by MominRaza** [comment]
Related update: https://github.com/Huachao/vscode-restclient/pull/1422

This discussion is addressed by the new `@set` directive, which lets you capture response values (`response.headers.*` / `response.body.*`) into shared runtime variables and reuse them across `.http` files.
